plugins/tiddlywiki/freelinks/text.js

```diff
@@ -1,23 +1,39 @@
 /*\
-title: $:/core/modules/widgets/text.js
+title: $:/plugins/tiddlywiki/freelinks/text.js
 type: application/javascript
 module-type: widget
 
-An override of the core text widget that automatically linkifies the text
+An override of the core text widget that automatically linkifies the text, with support for non-Latin languages like Chinese, prioritizing longer titles, skipping processed matches, excluding the current tiddler title from linking, and handling large title sets with Aho-Corasick algorithm and fixed chunking (100 titles per chunk). Includes optional persistent caching of Aho-Corasick automaton, controlled by $:/config/Freelinks/PersistAhoCorasickCache.
 
 \*/
+(function() {
```

Comment:
This function() wrapper is not needed anymore. We removed it from the whole repo